internal/syncronizedmap/syncronizedmap.go

```diff
@@ -0,0 +1,67 @@
+package syncronizedmap
+
+import (
+	"fmt"
+	"sync"
+)
+
+//go:generate mockgen -source=syncronizedmap.go -package=syncronizedmap -destination=mock_syncronizedmap.go
+
+type KernelOsDtkMapping interface {
+	NewNodeInfo(kernelVersion, osImage string)
+	NewImageStreamInfo(osImage, dtkImage string)
```

Comment:
I think the following names would be more explicit for a map:
```suggestion
	SetNodeInfo(kernelVersion, osImage string)
	SetImageStreamInfo(osImage, dtkImage string)
```
Or maybe
```suggestion
	SetKernelToOS(kernelVersion, osImage string)
	SetOSToDTKImage(osImage, dtkImage string)
```

Reply:
Prefer the first option. Fixed.